jms-client/src/main/java/no/vegvesen/ixn/ImageSink.java

```diff
@@ -0,0 +1,57 @@
+package no.vegvesen.ixn;
+
+import no.vegvesen.ixn.properties.MessageProperty;
+import org.apache.commons.io.FileUtils;
```

Comment:
Should not utse Commons IO classes, ewhen we can do the same using the standard classes.